chia/rpc/data_layer_rpc_client.py

```diff
@@ -1,11 +1,11 @@
-from typing import Any, Dict, List
+from typing import Any, Dict, List, Optional
 
 from chia.rpc.rpc_client import RpcClient
 from chia.types.blockchain_format.sized_bytes import bytes32
 
 
 class DataLayerRpcClient(RpcClient):
-    async def create_data_store(self) -> Dict[str, Any]:
+    async def create_data_store(self) -> Optional[Dict[str, Any]]:
```

Comment:
When would this return `None`?

https://github.com/Chia-Network/chia-blockchain/blob/94e961f41019be9a4c3a1154623a4a56c5708711/chia/rpc/data_layer_rpc_api.py#L59-L63

https://github.com/Chia-Network/chia-blockchain/blob/94e961f41019be9a4c3a1154623a4a56c5708711/chia/rpc/rpc_client.py#L44-L50

```suggestion
    async def create_data_store(self) -> Dict[str, Any]:
```